### PR TITLE
Bug 2090068: Lowering the label of the Name column in vm disks tab

### PR DESCRIPTION
**Pull request by DanaOrr**
Signed-off-by: Dana Orr <dorr@redhat.com>

## 📝 Description
The labels overlap each other as the page shrinks, so I fixed by lowering the label of the name.

## 🎥 Demo
Before:
image

After:
image

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/disk/tables/disk/DiskRow.tsx [hunk 1]
```diff
@@ -23,13 +23,13 @@ const DiskRow: React.FC<
   return (
     <>
       <TableData id="name" activeColumnIDs={activeColumnIDs}>
-        <Split hasGutter>
+        <Split className="disk-row-split">
```
Comment (avivtur) [inline src/views/virtualmachines/details/tabs/disk/tables/disk/DiskRow.tsx:26]: This should work nicely :) I want to suggest another possible solution: 
you could try replacing the `Split` component with `Stack` component from patternfly https://www.patternfly.org/v4/layouts/stack which I think will have a cleaner solution to this issue

Comment (yaacov) [reply src/views/virtualmachines/details/tabs/disk/tables/disk/DiskRow.tsx]: @avivtur please re-review

File: src/views/virtualmachines/details/tabs/disk/tables/disk/DiskRow.tsx [hunk 2]
```diff
@@ -23,18 +23,18 @@ const DiskRow: React.FC<
   return (
     <>
       <TableData id="name" activeColumnIDs={activeColumnIDs}>
-        <Split hasGutter>
+        <Stack>
           <SplitItem>
```
Comment (avivtur) [inline src/views/virtualmachines/details/tabs/disk/tables/disk/DiskRow.tsx:27]: if using the `Stack` component, you should replace also the `SplitItem` component with `StackItem` for consistency

File: src/views/virtualmachines/details/tabs/disk/tables/tables.scss [hunk 3]
```diff
@@ -6,3 +6,4 @@
 .icon-size-small {
   font-size: var(--pf-global--FontSize--sm);
 }
+
```
Comment (avivtur) [inline src/views/virtualmachines/details/tabs/disk/tables/tables.scss:9]: please remove this line

**Comment by hstastna** [comment]
@glekner @avivtur @metalice @pcbailey @yzamir @vojtechszocs please review

**Pull request merged**